recipes-bsp/device-tree/lmp-device-tree/portenta-x8/ov_som_x8h7.dts

```diff
@@ -61,7 +61,7 @@
 				pinctrl-0 = <&pinctrl_irq_x8h7>, <&pinctrl_gpio_x8h7>;
 				interrupt-parent = <&gpio1>;
 				interrupts = <9 IRQ_TYPE_LEVEL_LOW>;
-				spi-max-frequency = <50000000>;
```

Comment:
This change doesn't need to be here - the DTS specification specifies the maximum SPI frequency that the IMX8 is capable of, however the max. SPI frequency being used is limited in the driver, see [here](https://github.com/arduino/meta-partner-arduino/blob/main/recipes-kernel/kernel-modules/x8h7/x8h7_drv.c#L432).

Reply:
The

```spi-max-frequency```

property applies to the subdev node (in our case x8h7_drv.c). In practice there is no reason for the subdev driver to select a frequency lower than the maximum available. If we take spidev driver submodule as reference, you'll see that the spi-max-frequency is used to configure the default speed_hz in the spi_setup method. In the spidev driver it can be later changed via specific ioctl (but we don't handle such ioctl in our x8h7_drv.c)

- https://elixir.bootlin.com/linux/v5.10.93/source/drivers/spi/spi.c#L2019
- https://elixir.bootlin.com/linux/v5.10.93/source/drivers/spi/spidev.c#L456

Reply:
>property applies to the subdev node

okay, that makes sense :+1: